### ISSUE TITLE
Surface pen eraser doesn't work

### ISSUE THREAD

**Issue by Jos635** [xournalpp/xournalpp#1670]
**Affects versions :**
 - OS: Kubuntu 19.10
 - Version of Xournal++: tested on 6726c65db1efe6ed5363ff320b7764b8bea587c9 (latest commit in master)

**Describe the bug**
When using the eraser (the top side of the Surface Pen), it just draws more lines.

**To Reproduce**
1. Draw something
2. Turn the pen around and try to erase something

**Input Logs**
[evtest.txt](https://github.com/xournalpp/xournalpp/files/3964861/evtest.txt)
[debug_input.log](https://github.com/xournalpp/xournalpp/files/3964864/debug_input.log)
[debug_gdk_input.log](https://github.com/xournalpp/xournalpp/files/3964862/debug_gdk_input.log)
[xinput.txt](https://github.com/xournalpp/xournalpp/files/3964863/xinput.txt)
[debug_gdk_input_expirmental.log](https://github.com/xournalpp/xournalpp/files/3964884/debug_gdk_input_expirmental.log)
[debug_input_experimental.log](https://github.com/xournalpp/xournalpp/files/3964885/debug_input_experimental.log)

These logs are of me drawing a line, then trying to use the eraser to remove the line.

I followed the instructions to generate the output logs, no additional debugging output seems to be generated when the experimental input system is turned off. I've attached logs for both.

I've also attached an `evtest` log of using the eraser, which shows that touching the screen using the eraser reports two EV_KEY events:

```
Event: time 1576408816.739297, -------------- SYN_REPORT ------------
Event: time 1576408816.745546, type 1 (EV_KEY), code 320 (BTN_TOOL_PEN), value 1
Event: time 1576408816.745546, -------------- SYN_REPORT ------------
Event: time 1576408816.755059, type 1 (EV_KEY), code 321 (BTN_TOOL_RUBBER), value 1
```

**Expected behavior**
Using the eraser side of the Surface Pen should erase whatever is under the pen.

**Additional context**
I've configured all stylus "buttons" to be erasers. This issue happens with both the newest Surface Pen and the previous generation (Surface Pro 4).

**Comment by LittleHuba** [comment]
Listening for the “tool-changed” signal might solve this.

**Comment by chron-isch** [comment]
I have the same problem with ArchLinux on my Surface Go.

**Comment by tomasiser** [comment]
Turning the pen around and erasing also does not work with HP Spectre pen. It just draws more lines instead of erasing them.

**Comment by Tall-Lobster** [comment]
I created an issue as it turns out on Windows 10 the pen nor touchscreen are detected as input devices in xournal++
https://github.com/xournalpp/xournalpp/issues/1751

**Comment by Jos635** [comment]
This is now working correctly for me. Thank you to whoever fixed this!

**Comment by testbughub** [comment]
I've got this same issue.   
The side button won't work either.  
``evtest`` recognizes everything though.
Not sure if it's because I'm using bionic or something.  

OS: Linux Mint 19.3
Version of Xournal++: I tried both xournalpp-1.0.20-Ubuntu-bionic-x86_64.deb and through the PPA.
Device: Microsoft Surface 4 Pro Pen (with kernel from https://github.com/linux-surface/linux-surface)

I'm not really sure what more to add, but let me know if I should attach any logs or anything else.
